.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,24 @@
+name: GitHub Actions Demo
+on: [push, pull_request]
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
```

Comment:
There's a risk something could break when just using latest, so I would suggest using a specific version, for example ubuntu-20.04

Reply:
:+1: